### ISSUE TITLE
Issue while doing NPM install

### ISSUE THREAD

**Issue by valera-rozuvan** [jridgewell/set-array#1]
I am getting the following warning:

```
npm WARN deprecated @jridgewell/set-array@1.0.0: incorrect UMD name
```

when installing your NPM module.

I have the following versions:

```
node - v16.14.2
npm  - 8.8.0
```

**Comment by jridgewell** [comment]
I had another report at https://github.com/ampproject/remapping/issues/173, which [tracked it down](https://github.com/ampproject/remapping/issues/173#issuecomment-1111069759) to `@jridgewell/gen-mapping` v0.1.0 pinning `@jridgewell/set-array` to v1.0.0 (which has been deprecated due to a bug). I published `@jridgewell/gen-mapping` v0.1.1 (and now v0.2.0) which fixes the dependency range. Please try running `npm update` to resolve the issue.

**Comment by valera-rozuvan** [comment]
Thank you @jridgewell !! Issue resolved.
